src/reanimated2/Bezier.ts

```diff
@@ -52,16 +52,16 @@ export function Bezier(
     aX: number,
     aA: number,
     aB: number,
-    mX1: number,
-    mX2: number
+    innerMX1: number,
+    innerMX2: number
```

Comment:
Instead of renaming the arguments here, can we move the function to outer scope? Looks like we don't use values from closure inside this function.